OrbitGl/ModulesDataView.h

```diff
@@ -28,15 +28,18 @@ class ModulesDataView : public DataView {
   bool HasRefreshButton() const override { return true; }
   void OnRefreshButtonClicked() override;
 
-  void SetProcess(ProcessData* process);
+  // TODO(antonrohr) make const as soon as process_ is gone
```

Comment:
don't get the comment. the param is already const, and a const setter sounds strange. Can you rephrase, if it is still relevant?

Reply:
This comment does not make any sense. I removed it